src/components/AttachmentPicker/index.native.js

```diff
@@ -27,11 +27,14 @@ const propTypes = {
 /**
   * See https://github.com/react-native-image-picker/react-native-image-picker/#options
   * for ImagePicker configuration options
+  * maxWidth and maxHeight to avoid android crash when with large images
```

Comment:
Please add more details. What is the effect of adding these?

Reply:
> Please add more details. What is the effect of adding these?

done,

_______________




> @ahmdshrif Could you please add videos for all platforms? Videos should cover Image zooming. Also, please update the issue description. ![image](https://user-images.githubusercontent.com/24370807/149484068-bbc53556-4576-4158-83d7-1def323f7f58.png)
> 
> Should be `$ https://github.com/Expensify/App/issues/6836`

I update android and ios videos with zoom. 
note I only change files with extension .native.js 
so it only affects android and ios. 
also, this issue is only on android.

Reply:
Well, the description is still wrong. The template must be saying to add only the link to the issue. 

Could you please replace `$ GH_LINK` with `$ https://github.com/Expensify/App/issues/6836`

Reply:
ok, I change the link,
and describe the changes more.